### ISSUE TITLE
Bot-Tracker Bug: Incorrect Data is Displayed When Page is Reopened After Device Wakes From Sleep Mode

### ISSUE THREAD

**Issue by propertunist** [MattyIce/bottracker#3]
I just found that the amounts showing for the 'Total Bids' for a variety of bots were higher than their Vote Value and wondered if that was because I had just opened my PC from Sleep mode - where the bot tracker page had been open in the browser before the device went to sleep. It appears that the algorithm on the bot-tracker page doesn't account for these situations, so the data on the page was wrong. After I refreshed the page, the correct data was displayed.

To (probably) recreate:

1. Open Bot Tracker.
2. Put device to sleep.
3. Come back 30+ minutes later.
4. Wake up device.
5. There's a good chance that the displayed data will be different to what displays after a page refresh.

![bottracker](https://i.imgur.com/CM9LuN0.png)

<br /><hr/><em>Open Source Contribution posted via <a href="https://utopian.io/utopian-io/@ura-soul/bot-tracker-bug-incorrect-data-is-displayed-when-page-is-reopened-after-device-wakes-from-sleep-mode">Utopian.io</a></em><hr/>

**Comment by hbheiner** [comment]
yes wrong data... sometimes, the site show me opportunity about reward of 600% and more if use some bot... but the reward come often to negative balance...

and what about this bidhistory?
this show nothing
